security-privacy-questionnaire.md

```diff
@@ -1,24 +1,24 @@
+https://www.w3.org/TR/2021/NOTE-security-privacy-questionnaire-20211216/
 
 ### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
 
 This feature exposes files and directories the user explicitly selects to share with web sites with those web sites. This feature doesn't expose any more information than is already exposed via `<input type=file>` and `<input type=file webkitdirectory>` today.
 
 However this feature does make it possible for browsers to extend the time web sites have access to this information. I.e. permission grants and handles received through this API could be persisted giving web sites access to the same files/directories on disk when a user later returns to the website (or when a service worker for the origin is processing an event). At least for the Chrome implementation, we're only planning on having these grants persist for installed PWAs. On the drive-by web, access will only be as long as the web site is open, requiring re-prompting in subsequent visits.
 
-### 2.2. Is this specification exposing the minimum amount of information necessary to power the feature?
+### 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
 
 Yes, we're only exposing files and directories explicitly selected by the user. Of course, web sites could ask for access to a directory when all it needs is access to some files, but the same is already true today. At least as far as exposing information to web sites is concerned, this API doesn't expose any more information than existing APIs do today.
 
-### 2.3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+### 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
 
 No data is exposed without the user explicitly choosing what files or directories to expose to the web site.
 
-### 2.4. How does this specification deal with sensitive information?
+### 2.4. How do the features in your specification deal with sensitive information?
 
 No data is exposed without the user explicitly choosing what files or directories to expose to the web site, so only sensitive data that the user explicitly decides to share via this API will be shared. Furthermore, this API is only exposed in secure contexts, and third-party iframes (i.e. iframes that are cross origin from the top-level frame) won't be able to show pickers or permission prompts and can only access data they were already granted access to from a top-level same origin frame.
 
-### 2.5. Does this specification introduce new state for an origin that persists across browsing sessions?
+### 2.5. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
```

Comment:
This question gives IndexedDB as an example... which suggests the BucketFS should be included here. Or it might be worth creating a separate questionnaire on the whatwg spec?... Anyways, let's just add a TODO for now?

https://www.w3.org/TR/2021/NOTE-security-privacy-questionnaire-20211216/#persistent-origin-specific-state